api/src/Controller/Stores/DeleteProduct.php

```diff
@@ -0,0 +1,43 @@
+<?php declare(strict_types=1);
+
+namespace App\Controller\Stores;
+
+use App\Controller\BaseController;
+
+use Atlas\Query\Delete;
+
+use Slim\Http\Request;
+use Slim\Http\Response;
+
+class DeleteProduct extends BaseProduct
+{
+
+
+    public function buildResource(Request $request, Response $response, $params): array
+    {
+        if (!$_SESSION['IS_ADMIN']) {
```

Comment:
Generally I have been liking using RBAC methods here..
`        if (\ciab\RBAC::havePermission('api.delete.store'))`
Relying on `IS_ADMIN` is not the direction I think we should keep going.

Reply:
I agree, but I also don't want RBAC to get too crazy. It will be unmanageable if it continues to be based on fine-grained permissions like that. RBAC is about ROLE.

Reply:
This is in need of a broader conversation.  Here directly because I really dislike the _SESSION['IS_ADMIN'] and have been actively working to remove it everything.  so ADDING instances makes me unhappy.